eclass/python-utils-r1.eclass

```diff
@@ -678,12 +681,12 @@ python_newexe() {
 	(
 		dodir "${wrapd}"
 		exeinto "${d}"
-		newexe "${f}" "${newfn}" || die
+		newexe "${f}" "${newfn}" || return ${?}
 	)
 
 	# install the wrapper
 	_python_ln_rel "${ED%/}"/usr/lib/python-exec/python-exec2 \
-		"${ED%/}/${wrapd}/${newfn}" || die
+		"${ED%/}/${wrapd}/${newfn}" || die "${die_args[@]}" || return ${?}
```

Comment:
Not sure if we actually want nonfatal here. `newexe` can fail when input does not exist which makes sense for some nonfatal. This can only fail on write errors or internal failures.